cardpresent/src/main/java/com/fattmerchant/omni/Omni.kt

```diff
@@ -100,16 +100,21 @@ open class Omni internal constructor(internal var omniApi: OmniApi) {
                 mobileReaderDetails.anywhereCommerce?.let {
                     mutatedArgs["awc"] = it
                 }
+
+                // In case of Missing reader credentials
+                if (mutatedArgs["awc"] == null && mutatedArgs["nmi"] == null) {
+                    initialized = true
+                    error(OmniException("Could not get reader settings", "Your account does not have mobile reader credentials"))
+                    return@launch
+                }
```

Comment:
@omar-salah-elboredy We also need to account for when the credentials are blank, not just null